modules/administration-guide/partials/proc_restoring-a-backed-up-instance-of-an-earlier-version-of-che.adoc

```diff
@@ -0,0 +1,49 @@
+
+[id="restoring-a-backed-up-instance-of-an-earlier-version-of-{prod-id-short}_{context}"]
+= Restoring a backed up instance of an earlier version of {prod-short}
+
+You can restore a {prod-short} instance from a backup snapshot that was taken in an earlier version of {prod-short}, and you can roll back {prod-short} to that earlier version.
+
+include::snip_warning-backup-snapshots-are-bound-to-specific-cluster.adoc[]
+
+.Prerequisites
+
+* Installed `{prod-cli}`.
+* Backup snapshot of a {prod-short} instance of an earlier version of {prod-short} on any of the following:
+** xref:backups-of-che-instances-to-an-sftp-backup-server.adoc[External SFTP backup server]
+** xref:backups-of-che-instances-to-amazon-s3.adoc[Amazon S3]
+** xref:backups-of-che-instances-to-a-rest-backup-server.adoc[External REST backup server]
+** xref:backups-of-che-instances-to-the-internal-backup-server.adoc[Internal backup server]
+* Backup snapshot ID, which can be found in:
+** Command-line output when creating a backup
+** Custom resource status
```

Comment:
Not clear which custom resource.
What do you think about `Status of a CheClusterBackup custom resource`?